server/repository/src/db_diesel/system_log_row.rs

```diff
@@ -17,21 +18,29 @@ table! {
     }
 }
 
-#[derive(DbEnum, Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(DbEnum, Default, Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Display)]
+#[cfg_attr(test, derive(strum::EnumIter))]
 #[DbValueStyle = "SCREAMING_SNAKE_CASE"]
 pub enum SystemLogType {
+    #[default]
     ProcessorError,
+    LedgerFixError,
+    LedgerFix,
```

Comment:
LedgerFix would be used when we actually record info about the ledger fix we are doing